rmgpy/statmech/conformer.pyx

```diff
@@ -384,7 +384,8 @@ cdef class Conformer(RMGObject):
         elif pivots[0] in top1 and pivots[1] in top1:
             raise ValueError('Both pivot atoms included in top; you must specify only one pivot atom that belongs'
                              ' with the specified top.')
-
+        elif 0 in top1:
+            raise ValueError('Top must be zero indexed.')
```

Comment:
Should the message say "should be one-indexed" instead?

Reply:
XD...yes done